code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -171,11 +173,296 @@
 	allowed = list(/obj/item/melee/sickly_blade, /obj/item/forbidden_book)
 	hoodtype = /obj/item/clothing/head/hooded/cult_hoodie/eldritch
 	// slightly better than normal cult robes
-	armor = list("melee" = 50, "bullet" = 50, "laser" = 50,"energy" = 50, "bomb" = 35, "bio" = 20, "rad" = 0, "fire" = 20, "acid" = 20)
+	armor = list("melee" = 50, "bullet" = 50, "laser" = 50,"energy" = 50, "bomb" = 40, "bio" = 20, "rad" = 0, "fire" = 25, "acid" = 25, "stamina" = 50)
 
 /obj/item/reagent_containers/glass/beaker/eldritch
 	name = "flask of eldritch essence"
 	desc = "Toxic to the close minded. Healing to those with knowledge of the beyond."
 	icon = 'icons/obj/eldritch.dmi'
 	icon_state = "eldrich_flask"
 	list_reagents = list(/datum/reagent/eldritch = 50)
+
+/obj/item/artifact
+	name = "strange figurine"
+	desc = "A stone statuette of some sort."
+	var/inUse = FALSE
+	var/deity = 0
+	var/godname = "C'Thulhu"
+	var/activated = FALSE
+	var/ashes = FALSE
+	icon = 'icons/obj/wizard.dmi'	//temporary
```

Comment:
temporary

Reply:
Ah yes. This is where spriters are supposed to come in?